configure

```diff
@@ -1779,7 +1773,6 @@ if test "$x11_include" = "not found"; then
       if test $dir = /usr/lib; then
         x11_link="-lX11"
       else
-        x11_libs="-L$dir"
```

Comment:
Edited in 2012-02-01. Variable `x11_libs` can not be found with git grep. Probably here should be `x11_link`, which is defined 4 lines below.

Reply:
This variable was introduced in 296fc0547052da47bf0e983cab1ce173fa1e8882 when olabl was merged and was necessary for detection of Tcl/Tk - it should have been removed in e82104a755463d481667650ba4f00de535048f39.